### PR TITLE
Removed feature flag for check-in insurance page.

### PR DESCRIPTION
**Pull request by brianseek**
## Description of change

Removes un-used feature flag for insurance page in the check-in application.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#42133

## Things to know about this PR

Related vets-website PR: https://github.com/department-of-veterans-affairs/vets-website/pull/21439

### CODE REVIEW

**Comment by brianseek** [comment]
On hold until vets-website PR is merged and in production.

**Review by randomsync — APPROVED** [review]

**Pull request merged**
